packaging/Makefile

```diff
@@ -0,0 +1,18 @@
+NAME    := libfabric
+SRC_EXT := gz
+SOURCE   = https://github.com/ofiwg/$(NAME)/archive/v$(VERSION).tar.$(SRC_EXT)
+
+OSUSE_HPC_REPO := https://download.opensuse.org/repositories/science:/HPC
+LEAP_42_HPC_REPO := $(OSUSE_HPC_REPO)/openSUSE_Leap_42.3/
+
+ifeq ($(DAOS_STACK_LEAP_42_GROUP_REPO),)
+LEAP_42_REPOS  := $(LEAP_42_HPC_REPO)
+endif
+ifeq ($(DAOS_STACK_SLES_12_GROUP_REPO),)
+SLES_12_REPOS := $(LEAP_42_HPC_REPO)
+endif
```

Comment:
I think we can get rid of these lines now.  We soooo don't support those distros any more.